hydra-node/test/Hydra/ModelSpec.hs

```diff
@@ -224,15 +224,11 @@ prop_checkModel actions =
         -- now a "more active" simulated chain which ticks away and not simply
         -- detects a deadlock if we wait for infinity. Maybe cancelling the
         -- simulation's 'tickThread' and wait then could work?
-        run $ lift waitForADay
```

Comment:
Actually, we don't wait a day in `IOSim`. I don't remember why we put that wait here, perhaps as a way to reach some quiescent state. A `threadDelay` in IOSim means that any other non waiting thread will be scheduled before the waiting thread so the number in itself does not matter much unless in relationship with other delays or timeouts.

Reply:
I have pushed a change which reduces the delay to one minute, should be enough. We could also drop that test as I don't think it adds value anymore